src/components/Buzz/Dashboard/BuzzDashboardOverview.tsx

```diff
@@ -50,120 +51,86 @@ export const BuzzDashboardOverview = ({ accountId }: { accountId: number }) => {
   const { classes, theme } = useBuzzDashboardStyles();
   // Right now, sadly, we neeed to use two separate queries for user and generation transactions.
   // If this ever changes, that'd be awesome. But for now, we need to do this.
-  const mainBuzzTransactions = useBuzzTransactions(accountId, 'user');
-  const generationBuzzTransactions = useBuzzTransactions(accountId, 'generation');
   const [transactionType, setTransactionType] = React.useState<'user' | 'generation'>('user');
+  const transactionData = useBuzzTransactions(accountId, transactionType);
+  const [reportFilters, setReportFilters] = React.useState<GetTransactionsReportSchema>({
+    window: 'day',
+    accountType: ['User', 'Generation'],
+  });
+  const { report, isLoading, isFetching } = useTransactionsReport(reportFilters, {
+    enabled: true,
+  });
 
-  const transactions = useMemo(() => {
-    const transactions =
-      transactionType === 'user'
-        ? mainBuzzTransactions.transactions
-        : generationBuzzTransactions.transactions;
+  const isLoadingReport = isLoading || isFetching;
```

Comment:
Should it be `isRefetching` instead?
```suggestion
  const isLoadingReport = isLoading || isRefetching;
```